geometry_msgs/Pose2D current_mobile_state
geometry_msgs/Pose2D target_mobile_pose
mobile_motion_planner/Obstacle2D[] Obstacles2D
---
mobile_motion_planner/MobileTrajectory mobile_trajectory
